### PR TITLE
Updated solutions list on using REMOTE_USER how-to.

### PR DESCRIPTION
**Pull request by AfiMaameDufie**
#### Trac ticket number

ticket-XXXXX

#### Branch description
Updated the mod_authnz_ldap link which formerly pointed to the documentation for an unsupported version to the current version.
Removed the outdated Cosign link(the software is no longer being maintained and the link currently points to a dead site).

#### Checklist
- [ ] This PR targets the `main` branch. 
- [ ] The commit message is written in past tense, mentions the ticket number, and ends with a period.
- [ ] I have checked the "Has patch" ticket flag in the Trac system.
- [ ] I have added or updated relevant tests.
- [ ] I have added or updated relevant docs, including release notes if applicable.
- [ ] I have attached screenshots in both light and dark modes for any UI changes.

### CODE REVIEW

**Review by thibaudcolas — APPROVED** [review]
[Looking good](https://django--18704.org.readthedocs.build/en/18704/howto/auth-remote-user.html) @AfiMaameDufie! 🙌 thank you for giving this a go.

**Comment by AfiMaameDufie** [comment]
> [Looking good](https://django--18704.org.readthedocs.build/en/18704/howto/auth-remote-user.html) @AfiMaameDufie! 🙌 thank you for giving this a go.

Thank you too for the support

File: docs/howto/auth-remote-user.txt [hunk 1]
```diff
@@ -6,12 +6,11 @@ This document describes how to make use of external authentication sources
 (where the web server sets the ``REMOTE_USER`` environment variable) in your
 Django applications.  This type of authentication solution is typically seen on
 intranet sites, with single sign-on solutions such as IIS and Integrated
-Windows Authentication or Apache and `mod_authnz_ldap`_, `CAS`_, `Cosign`_,
+Windows Authentication or Apache and `mod_authnz_ldap`_, `CAS`_,
```
Comment (felixxm) [inline docs/howto/auth-remote-user.txt:9]: Rewrap at 79 chars after `Cosign` removal, i.e.
```
Windows Authentication or Apache and `mod_authnz_ldap`_, `CAS`_, `WebAuth`_,
`mod_auth_sspi`_, etc.
```

Comment (AfiMaameDufie) [reply docs/howto/auth-remote-user.txt:9]: Okay. Fixing it now.

File: docs/howto/auth-remote-user.txt [hunk 2]
```diff
@@ -6,8 +6,7 @@ This document describes how to make use of external authentication sources
 (where the web server sets the ``REMOTE_USER`` environment variable) in your
 Django applications.  This type of authentication solution is typically seen on
 intranet sites, with single sign-on solutions such as IIS and Integrated
-Windows Authentication or Apache and `mod_authnz_ldap`_, `CAS`_,
-`WebAuth`_, `mod_auth_sspi`_, etc.
+Windows Authentication or Apache and `mod_authnz_ldap`_, `CAS`_, `WebAuth`_, `mod_auth_sspi`_, etc.
```
Comment (felixxm) [inline docs/howto/auth-remote-user.txt:9]:
```suggestion
Windows Authentication or Apache and `mod_authnz_ldap`_, `CAS`_, `WebAuth`_,
`mod_auth_sspi`_, etc.
```

**Review by felixxm — APPROVED** [review]

**Comment by felixxm** [comment]
@AfiMaameDufie Welcome aboard :boat:

**Comment by AfiMaameDufie** [comment]
> @AfiMaameDufie Welcome aboard ⛵

Thank youu

**Review by sarahboyce — APPROVED** [review]
Thank you @AfiMaameDufie :rocket:

**Comment by thibaudcolas** [comment]
Nice one @AfiMaameDufie ☺️

**Pull request merged**